WordPress/src/main/java/org/wordpress/android/ui/photopicker/PhotoPickerFragment.kt

```diff
@@ -26,6 +27,10 @@ import org.wordpress.android.ui.media.MediaBrowserType
 import org.wordpress.android.ui.media.MediaPreviewActivity
 import org.wordpress.android.ui.mediapicker.MediaPickerViewModel.ProgressDialogUiModel
 import org.wordpress.android.ui.mediapicker.MediaPickerViewModel.ProgressDialogUiModel.Visible
+import org.wordpress.android.ui.photopicker.PhotoPickerViewModel.ActionModeUiModel
+import org.wordpress.android.ui.photopicker.PhotoPickerViewModel.ActionModeUiModel.Hidden
+import org.wordpress.android.ui.photopicker.PhotoPickerViewModel.PermissionsRequested.CAMERA
+import org.wordpress.android.ui.photopicker.PhotoPickerViewModel.PermissionsRequested.STORAGE
```

Comment:
Minor (🔍): As per my previous [such comment](https://github.com/wordpress-mobile/WordPress-Android/pull/17393#discussion_r1010454204) please consider removing this extra imports and revert them back to how it was.

PS: When you commit a change please be careful on other changes, like import updates and revert those before committing.